content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/stock.rst

```diff
@@ -0,0 +1,112 @@
+============
+Stock report
+============
+
+Use the stock report in Odoo *Inventory* for a detailed list of all stored products, including those
+reserved, purchased and in transit, as well as those delivered to customers.
+
+.. note::
+   The reporting feature only accessible to users with :doc:`admin access
+   <../../../../general/users/access_rights>`.
+
+To access the stock report, go to :menuselection:`Inventory app --> Reporting --> Stock`.
+
+.. image:: stock/stock-report.png
+   :align: center
+   :alt: Show the stock report, accessible by going to Inventory > Reporting > Stock.
+
+Navigate the stock report
+=========================
+
+On the stock report, the left sidebar includes :guilabel:`Warehouses` to filter products by specific
+warehouses and :guilabel:`Category`, to show products within a selected product category.
+
+In the report itself, the columns represent:
+
+- :guilabel:`Product`: name of the product.
+- :guilabel:`Unit Cost`: average inventory valuation per unit, adjusted based on purchasing costs.
+- :guilabel:`Total Value`: Total inventory valuation of the product, calculated by multiplying unit
+  cost by on-hand quantity.
+
+.. seealso::
+   :doc:`../inventory_valuation/using_inventory_valuation`
```

Comment:
I think I am confused- is the cost, the cost to purchase or manufacture? And then the value is different? I just think they can be explained a touch further since I am not 100% sure the difference between the two.

Reply:
Added some more details! thank you (: